catchup/service.go

```diff
@@ -124,6 +124,7 @@ func MakeService(log logging.Logger, config config.Local, net network.GossipNode
 func (s *Service) Start() {
 	s.done = make(chan struct{})
 	s.ctx, s.cancel = context.WithCancel(context.Background())
+	atomic.SwapUint32(&s.initialSyncNotified, 0)
```

Comment:
'swap' is the wrong operation here since you're throwing away the swapped-out value. `atomic.Store*()` ?